### PR TITLE
first changes for enterprise dashboard

### PR DESCRIPTION
**Pull request by karst3n**

### CODE REVIEW

**Review by pranavsingh321 — APPROVED** [review]
looks good to me.

File: api-ref/source/v1/organizations.inc
Comment (hodigy) [inline api-ref/source/v1/organizations.inc:1]: Docs is in place

**Review by hodigy — APPROVED** [review]

**Comment by maltekluever** [comment]
okay!

**Review by maltekluever — APPROVED** [review]
okay!

**Review by pranavsingh321 — APPROVED** [review]
Looks good

**Comment by karst3n** [comment]
gate

**Comment by vladimirhasko** [comment]
regate

**Comment by vladimirhasko** [comment]
recheck

**Review by vladimirhasko — APPROVED** [review]

**Comment by vladimirhasko** [comment]
gate

**Pull request merged**